package.json

```diff
@@ -35,5 +35,5 @@
   "bugs": {
     "url": "https://github.com/mapbox/makizushi/issues"
```

Comment:
If this is now focused on our fork instead of the original, should we use https://phabricator.wikimedia.org/tag/kartotherian/ here?